releasenotes/notes/logarithmic_mapper-f655fc66d319ba08.yaml

```
@@ -0,0 +1,6 @@
+features:
+  - |
+    Adds :class:`qiskit_nature.mappers.second_quantization.logarithmic_mapper`.
```

```suggestion
    Adds :class:`qiskit_nature.mappers.second_quantization.LogarithmicMapper`.
```
This needs to refer to the class itself not the module that its in (the release note showed the term highlighted but it was not a link). I guess we want to show the full path here for the context where the class is, that's fine I think.